lib/scan/views/scan_qr_screen.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mobile_test/base/result_state.dart';
+import 'package:mobile_test/scan/cubit/scan_qr_cubit.dart';
+import 'package:qr_code_scanner/qr_code_scanner.dart';
+
+class ScanQrScreen extends StatelessWidget {
+  const ScanQrScreen({Key? key}) : super(key: key);
+
+  static Widget init() {
+    return BlocProvider(
+      create: (_) => ScanQrCubit(),
+      child: const ScanQrScreen(),
+    );
+  }
```

Comment:
question: Why did you choose a static method over instantiating the class itself directly?